projects/indicators/src/modules/text-highlight/text-highlight.directive.ts

```diff
@@ -30,12 +30,27 @@ export class SkyTextHighlightDirective
    * Specifies the text to highlight.
    */
   @Input()
-  public skyHighlight: string;
+  public set skyHighlight(value: string | string[]) {
+    value = value || [];
+    this._skyHighlight = [];
+    if (typeof value === 'string') {
```

Comment:
It might be safer to validate that the value is an array (`Array.isArray(value)`) than it is to assume that any non-string value is an array. We might have consumers supplying numbers or boolean values, for example, and it just works because JavaScript is good at coercing them into strings.

Reply:
Done.

Reply:
My original suggestion was to do this, which is a little different than what you've implemented. This keeps the original behavior mostly intact and only adds branching logic where the value is an array.

```
if (Array.isArray(value)) {
  // Do array stuff
} else {
  // Do anything else stuff
}
```

Reply:
Updated.